test/gosec.spawn.test.js

```diff
@@ -1,36 +1,41 @@
 // Copyright 2018 VMware, Inc.
 // SPDX-License-Identifier: BSD-2-Clause
 
-const gosec = require('../gosec/gosec')
 const fs = require('fs-extra')
 
-describe('Spawn gosec tests', () => {
-  test('Empty input for gosec', async () => {
-    const gosecResult = await gosec('', [])
-    expect(gosecResult).toBeFalsy()
-  })
```

Comment:
Do we need to remove to remove the test were we give empty array as input file ?

Reply:
Fixed.